src/lib/publish.js

```diff
@@ -38,7 +42,23 @@ async function publishBatch({ queue, messages, logger }) {
   }
 }
 
+/**
+ * publish a message to a notification topic
+ * @param topic {string} - topic to publish message to, e.g. an SNS ARN
+ * @param message - message to send
+ * @param {SNSClient} client - SNS client to issue command to
+ * @returns Promise<void>
+ */
+async function notify({ client = snsClient, message, topic }) {
+  const command = new PublishCommand({
+    Message: (typeof message === 'string') ? message : JSON.stringify(message),
```

Comment:
the function is been called only with `object`, we don't need to check the type. Also, jsdoc should be updated properly

Reply:
done!